www/js/viewer.js

```diff
@@ -215,24 +214,42 @@ const createCards = (rows, LineID) => {
   const cards = [];
   const lines = [];
   const shabad = {};
-  // empty the object from previous values
+
   Object.keys(rows).forEach((key) => {
     row = rows[key];
     lines.push(row.ID);
-    // const gurmukhiShabads = row.GurmukhiBisram.split(' ');
     const gurmukhiShabads = row.Gurmukhi.split(' ');
+    if (row.Visraam) {
+      const visraams = JSON.parse(row.Visraam);
```

Comment:
should this should be in a `try`? just in case a downloaded db pushes something wrong or the contract changes, we wouldn't want the app to break immediately. cc @navdeepsinghkhalsa

Reply:
Oh yes, it should be. Thanks. 🙏